Run the auto-installation script bash <(curl -Ss https://my-netdata.io/kickstart.sh)  failed

Run the auto-installation script bash <(curl -Ss https://my-netdata.io/kickstart.sh)  failed


  GEN      netdata
/usr/bin/ld: externaldeps/libwebsockets/libwebsockets.a(unix-caps.c.o): in function `lws_plat_drop_app_privileges':
unix-caps.c:(.text+0x23e): undefined reference to `cap_get_proc'
/usr/bin/ld: unix-caps.c:(.text+0x25a): undefined reference to `cap_set_flag'
/usr/bin/ld: unix-caps.c:(.text+0x262): undefined reference to `cap_set_proc'
/usr/bin/ld: unix-caps.c:(.text+0x282): undefined reference to `cap_free'
/usr/bin/ld: unix-caps.c:(.text+0x2d8): undefined reference to `cap_get_proc'
/usr/bin/ld: unix-caps.c:(.text+0x2f1): undefined reference to `cap_set_flag'
/usr/bin/ld: unix-caps.c:(.text+0x2f9): undefined reference to `cap_set_proc'
/usr/bin/ld: unix-caps.c:(.text+0x319): undefined reference to `cap_free'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:4065: netdata] Error 1
make[2]: Leaving directory '/tmp/netdata-kickstart-CnwrS4/netdata-v1.20.0-303-g007f27cd'
make[1]: *** [Makefile:5124: all-recursive] Error 1
make[1]: Leaving directory '/tmp/netdata-kickstart-CnwrS4/netdata-v1.20.0-303-g007f27cd'
make: *** [Makefile:2788: all] Error 2
 FAILED

 FAILED

 ABORTED  netdata-installer.sh exited with error

it's entire the same issue as @xgxdmx Mine looks like this: 
externaldeps/libwebsockets/libwebsockets.a(unix-caps.c.o): In function `lws_plat_drop_app_privileges':
unix-caps.c:(.text+0x381): undefined reference to `cap_get_proc'
unix-caps.c:(.text+0x399): undefined reference to `cap_set_flag'
unix-caps.c:(.text+0x3a1): undefined reference to `cap_set_proc'
unix-caps.c:(.text+0x3c1): undefined reference to `cap_free'
unix-caps.c:(.text+0x411): undefined reference to `cap_get_proc'
unix-caps.c:(.text+0x42d): undefined reference to `cap_set_flag'
unix-caps.c:(.text+0x435): undefined reference to `cap_set_proc'
unix-caps.c:(.text+0x455): undefined reference to `cap_free'
collect2: error: ld returned 1 exit status
Makefile:3845: recipe for target 'netdata' failed
make[2]: *** [netdata] Error 1
make[2]: Leaving directory '/home/crn01/netdata'
Makefile:4899: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/crn01/netdata'
Makefile:2569: recipe for target 'all' failed
make: *** [all] Error 2
 FAILED

--- a/Makefile.am
+++ b/Makefile.am
@@ -642,6 +642,7 @@ if ENABLE_ACLK
 netdata_LDADD = \
     externaldeps/mosquitto/libmosquitto.a \
     externaldeps/libwebsockets/libwebsockets.a \
+    $(OPTIONAL_LIBCAP_LIBS) \
     $(NETDATA_COMMON_LIBS) \
     $(NULL)
 else

--- a/configure.ac
+++ b/configure.ac
@@ -583,7 +583,7 @@ if test "$enable_cloud" != "no"; then
 
     AC_MSG_CHECKING([if libwebsockets static lib is present])
     if test -f "externaldeps/libwebsockets/libwebsockets.a"; then
-        LWS_LIBS="-I externaldeps/libwebsockets/include"
+        LWS_CFLAGS="-I externaldeps/libwebsockets/include"
         HAVE_libwebsockets_a="yes"
     else
         HAVE_libwebsockets_a="no"
@@ -596,7 +596,7 @@ if test "$enable_cloud" != "no"; then
             AC_MSG_ERROR([agent-cloud-link can't be built without libcap. Disable it by --disable-cloud or enable libcap])
         fi
         if test "${with_libcap}" = "yes"; then
-            LWS_LIBS+=" -lcap"
+            LWS_CFLAGS+=" ${LIBCAP_CFLAGS}"
         fi
     fi
 
@@ -1230,7 +1230,7 @@ AC_SUBST([webdir])
 
 CFLAGS="${CFLAGS} ${OPTIONAL_MATH_CFLAGS} ${OPTIONAL_NFACCT_CFLAGS} ${OPTIONAL_ZLIB_CFLAGS} ${OPTIONAL_UUID_CFLAGS} \
     ${OPTIONAL_LIBCAP_CFLAGS} ${OPTIONAL_IPMIMONITORING_CFLAGS} ${OPTIONAL_CUPS_CFLAGS} ${OPTIONAL_XENSTAT_FLAGS} \
-    ${OPTIONAL_KINESIS_CFLAGS} ${OPTIONAL_PROMETHEUS_REMOTE_WRITE_CFLAGS} ${OPTIONAL_MONGOC_CFLAGS} ${LWS_LIBS}"
+    ${OPTIONAL_KINESIS_CFLAGS} ${OPTIONAL_PROMETHEUS_REMOTE_WRITE_CFLAGS} ${OPTIONAL_MONGOC_CFLAGS} ${LWS_CFLAGS}"
 
 CXXFLAGS="${CFLAGS} ${CXX11FLAG}"
 

